lib/rules/eol-last.js

```diff
@@ -71,15 +71,21 @@ module.exports = class EolLast extends Rule {
 
           let lastNode = node.body[node.body.length - 1];
 
-          if (lastNode.type !== 'TextNode') {
-            return;
-          }
-
           if (this.mode === 'fix') {
-            if (lastNode.chars.endsWith('\n') && this.config === 'never') {
+            if (
+              lastNode.type === 'TextNode' &&
+              lastNode.chars.endsWith('\n') &&
+              this.config === 'never'
+            ) {
               lastNode.chars = lastNode.chars.slice(0, -1);
-            } else if (!lastNode.chars.endsWith('\n') && this.config === 'always') {
+            } else if (
+              lastNode.type === 'TextNode' &&
+              !lastNode.chars.endsWith('\n') &&
+              this.config === 'always'
+            ) {
               lastNode.chars = `${lastNode.chars}\n`;
+            } else if (lastNode.type !== 'TextNode' && this.config === 'always') {
+              // TODO: add new TextNode with empty line to end of body
```

Comment:
Will update PR once I figure out how to create a new `TextNode` with `\n` to add to the end of the file body.